app/page/review/transaction.tsx

```diff
@@ -148,6 +101,8 @@ function UserOperationConfirmation(props: {
     markERC4337TransactionRejected
   } = useAction()
 
+  const isContract = tx.data !== "0x"
```

Comment:
Also `||` check `tx.to` has code or not with `provider.getCode`.

Reply:
[Also check if tx.to has code](https://github.com/pilagod/waallet-extension/pull/159/commits/94087cce23574dfe89adec998e45bdddb5c91346)